SPECS/kernel/kernel.spec

```diff
@@ -321,6 +333,7 @@ then
      fi
 fi
 %grub2_postun
+%systemd_postun cpupower.service
```

Comment:
this should be in a `%postun tools` scriptlet, not the base `%postun` scriptlet, right?